target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network

```diff
@@ -26,9 +29,30 @@ ipq60xx_setup_interfaces()
 	esac
 }
 
+ipq60xx_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
+
+	case $board in
+	linksys,mr7350)
+		label_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
```

Comment:
Is this a u-boot env like setup?

Reply:
The `devinfo` partition contains key=value data:
```
hexdump -C -n 48 /dev/mtd20
00000000  6d 66 67 5f 64 61 74 61  5f 76 65 72 73 69 6f 6e  |mfg_data_version|
00000010  3d 31 0a 6d 6f 64 65 6c  4e 75 6d 62 65 72 3d 4d  |=1.modelNumber=M|
00000020  58 38 35 30 30 0a 73 65  72 69 61 6c 5f 6e 75 6d  |X8500.serial_num|
00000030
```

Reply:
Yeah, sadly no luck with NVMEM currently then